css/src/tokens/themes.scss

```diff
@@ -212,15 +212,15 @@ $themes: (
 		'info-active': $palette-purple-30,
 		'info-box-shadow': $palette-purple-opacity-70,
 		'info-invert': $palette-black,
-		'warning-base': $palette-yellow-50,
-		'warning-background': $palette-yellow-80,
-		'warning-background-hover': $palette-yellow-50,
-		'warning-background-glow-high-contrast': $palette-yellow-80,
-		'warning-dark': $palette-yellow-10,
-		'warning-dark-hover': $palette-yellow-30,
-		'warning-hover': $palette-yellow-40,
-		'warning-active': $palette-yellow-30,
-		'warning-box-shadow': $palette-yellow-opacity-70,
+		'warning-base': $palette-orange-120,
+		'warning-background': $palette-orange-220,
```

Comment:
I _think_ this should be `$palette-orange-210`

![image](https://github.com/user-attachments/assets/215ca665-c901-4c6b-bb40-d403ac6fa03a)

![image](https://github.com/user-attachments/assets/362a2de2-2db0-4841-92aa-305c56c2d699)